adapters/repos/db/index.go

```diff
@@ -153,6 +153,16 @@ type Index struct {
 	lastBackup  atomic.Pointer[BackupState]
 }
 
+func (i *Index) GetShards() []*Shard {
+	var out []*Shard
```

Comment:
Consider `out := make(shard, 0, len(shards))` for improved efficiency.

Reply:
Can't take the length of a sync.Map